src/Stratis.Bitcoin/Connection/ConnectionManager.cs

```diff
@@ -368,8 +369,8 @@ public void RemoveNodeAddress(IPEndPoint ipEndpoint)
             }
 
             this.peerAddressManager.RemovePeer(ipEndpoint.MapToIpv6(), IPAddress.Loopback);
-            List<IPEndPoint> matchingAddNodes = this.ConnectionSettings.AddNode.Where(p => p.Match(ipEndpoint)).ToList();
-            matchingAddNodes.ForEach(m => this.ConnectionSettings.AddNode.Remove(m));
+            IEnumerable<IPEndPoint> matchingAddNodes = this.ConnectionSettings.AddNode.Where(p => p.Match(ipEndpoint));
+            foreach (IPEndPoint m in matchingAddNodes) this.ConnectionSettings.AddNode.Remove(m);
```

Comment:
I like this change, though you could put the body on the separated line as we always do with loops

Reply:
I agree try to put bodies in a new line especially when its an operation.